### ISSUE TITLE
Remove premium tax credit code from IRS section

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-us#4211]
I believe this is duplicative of the `gov/aca` code that @martinholmer created.

https://github.com/PolicyEngine/policyengine-us/tree/master/policyengine_us/parameters/gov/irs/credits/premium_tax_credit

**Comment by martinholmer** [comment]
But doesn't the OLD PTC code rely on this?
The NEW PTC code has been implemented only for California.

**Comment by MaxGhenis** [comment]
Yes, but I don't think we use the old PTC code anywhere. Is that right @martinholmer and @nikhilwoodruff? We will at some point expand the California code nationwide.

**Comment by martinholmer** [comment]
@MaxGhenis said:
> Yes, but I don't think we use the old PTC code anywhere.

My impression is that the OLD PTC code is widely used in Policyengine-US.
